src/pretix/plugins/banktransfer/templates/pretixplugins/banktransfer/pending.html

```diff
@@ -85,20 +79,55 @@
+                    <div id="banktransfer_qrcodes_bezahlcode"
+                        role="tabpanel"
+                        tabindex="0"
+                        hidden
+                        aria-labelledby="banktransfer_qrcodes_bezahlcode_tab"
+                    >
+                        <a aria-label="{% trans "Open BezahlCode in your banking app to start the payment process." %}" href="bank://singlepaymentsepa?name={{ settings.bank_details_sepa_name|urlencode }}&iban={{ settings.bank_details_sepa_iban }}&bic={{ settings.bank_details_sepa_bic }}&amount={{ amount|commadecimal }}&reason={{ code }}&currency={{ event.currency }}">
+                            <div role="figure" aria-labelledby="banktransfer_qrcodes_bezahlcode_tab banktransfer_qrcodes_label">
                                 <script type="text/plain" data-size="150" data-replace-with-qr data-desc="{% trans 'Scan this image with your banking app’s QR-Reader to start the payment process.' %}">bank://singlepaymentsepa?name={{ settings.bank_details_sepa_name|urlencode }}&iban={{ settings.bank_details_sepa_iban }}&bic={{ settings.bank_details_sepa_bic }}&amount={{ amount|commadecimal }}&reason={{ code }}&currency={{ event.currency }}</script>
-                            </a>
-                        </p>
+                            </div>
+                        </a>
                     </div>
                 {% endif %}
             </div>
+            <div id="banktransfer_qrcodes_tabs" role="tablist" aria-labelledby="banktransfer_qrcodes_label" class="blank-after btn-group">
+            {% if swiss_qrbill %}
+                <button
+                    class="btn btn-default"
+                    id="banktransfer_qrcodes_qrbill_tab"
+                    type="button"
+                    role="tab"
+                    aria-controls="banktransfer_qrcodes_qrbill"
+                    aria-selected="true"
+                    tabindex="-1">QR-bill</button>
+            {% endif %}
+            {% if eu_barcodes %}
+                <button
+                    class="btn btn-default"
+                    id="banktransfer_qrcodes_girocode_tab"
+                    type="button"
+                    role="tab"
+                    aria-controls="banktransfer_qrcodes_girocode"
+                    aria-selected="{{ swiss_qrbill|yesno:"false,true" }}"
+                    tabindex="-1">EPC-QR</button>
+                <button
+                    class="btn btn-default"
+                    id="banktransfer_qrcodes_bezahlcode_tab"
+                    type="button"
+                    role="tab"
+                    aria-controls="banktransfer_qrcodes_bezahlcode"
+                    aria-selected="false"
+                    tabindex="-1">BezahlCode</button>
+            {% endif %}
+            </div>
+            <p class="text-muted" id="banktransfer_qrcodes_label">
+                {% trans "Scan the qr-code with your banking app" %}
```

Comment:
Not new, but still wrong
```suggestion
                {% trans "Scan the QR code with your banking app" %}
```